estimagic/dashboard/utilities.py

```diff
@@ -0,0 +1,133 @@
+"""Helper functions for the dashboard."""
+import random
+import socket
+from contextlib import closing
+from pathlib import Path
+
+import bokeh.palettes
+from bokeh.models.widgets import Div
+from bokeh.plotting import figure
+
+
+def create_short_database_names(path_list):
+    """Generate short but unique names from each path for each full database path.
+
+    Args:
+        path_list (list): Strings or pathlib.Paths to the optimizations' databases.
+
+    Returns:
+        short_name_to_path (dict): Mapping from the new unique names to their full path.
+
+    Example:
+
+    >>> pl = ["bla/blubb/blabb.db", "a/b", "bla/blabb"]
+    >>> create_short_database_names(pl)
+    {'blubb/blabb': 'bla/blubb/blabb.db', 'b': 'a/b', 'bla/blabb': 'bla/blabb'}
+
+    """
+    no_suffixes = [Path(p).resolve().with_suffix("") for p in path_list]
```

Comment:
We prefer not to show the file extension in the overview because it looks nicer. In the documentation we wrote that the databases must have the file extension `.db` (even though this is never enforced). So only experienced database users would choose something different. Also, I would say that using `.db` and `.sql` to make a distinction between two databases is not a good idea in general and people who do it should run into problems now and then. 

Having said that, I'm not so sure that omitting the file extension is a good idea. Does anyone have strong preferences about this?

Reply:
No.